src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrent.cs

```diff
@@ -116,11 +110,9 @@ public override IEnumerable<DownloadClientItem> GetItems()
                 item.RemainingTime = TimeSpan.FromSeconds(torrent.Eta);
 
                 item.OutputPath = _remotePathMappingService.RemapRemoteToLocal(Settings.Host, new OsPath(torrent.SavePath));
-                
-                // At the moment there isn't an easy way to detect if the torrent has
-                // reached the seeding limit, We would need to check the preferences 
-                // and still not be completely sure if that torrent has a limit set for it
-                item.IsReadOnly = true;
+
+                // Avoid removing torrents that haven't reached their max ratio
+                item.IsReadOnly = config.MaxRatioEnabled && config.MaxRatio > torrent.ShareRatio;
```

Comment:
> sorry, are you saying that a user may want to unpause a completed torrent to prevent Sonarr from importing?

not prevent it from import, but prevent Sonarr from removing it. Another use-case (that seems likely to happen) is if a user increases the ratio required to pause the torrent, since Sonarr only knows about the global ratio it would treat it as finished even if it's still seeding because that torrent hasn't hit it's ratio requirement.